### PR TITLE
add since tag for 1.5

### PR DESCRIPTION
**Pull request by kailash-choudhary-thoughtspot**
Introduce version includes in for new APIs in October
https://thoughtspot.atlassian.net/browse/SCAL-97225

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -117,6 +117,7 @@ export interface EmbedConfig {
     /**
      * Calls the prefetch method internally when set to true
      * @default false
+     * @since 1.5.0_
```
Comment (akashmohapatra) [inline src/types.ts:120]: This will be a part of 1.4.0, hence need not have the since declaration

File: src/types.ts [hunk 2]
```diff
@@ -337,6 +338,7 @@ export enum HostEvent {
      *              eg. { selectedPoints: []}
      * @param columnGuid - a string guid of the column to drill by. This is optional,
      *                     if not provided it will auto drill by the configured column.
+     * @since 1.5.0_
```
Comment (akashmohapatra) [inline src/types.ts:341]: @ashubham Do you know if this goes in 1.4 or 1.5?

File: src/embed/pinboard.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ export interface PinboardViewConfig extends ViewConfig {
      * Setting this height helps resolves issues with empty pinboards and
      * other screens navigable from a pinboard.
      * @default 500
+     * @since 1.5.0_
```
Comment (ashubham) [inline src/embed/pinboard.ts:38]: _since 1.5.0_ drop the @

File: src/types.ts (hunk 2 above)
Comment (ashubham) [reply src/types.ts:340]: 1.5

**Review by ashubham — COMMENTED** [review]
We would also need to run `npm run docgen` to generate the new API docs.

**Pull request merged**